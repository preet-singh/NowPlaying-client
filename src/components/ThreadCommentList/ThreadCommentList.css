#ThreadCommentList h4 {
  background: black;
}

.thread-comment-list {
  text-align: center;
  background: black;
}


@media screen and (min-width: 768px){
  .thread-comment-list {
    position: absolute;
    top: 200px; 
    right: 50px;
    border: 1px solid white;
    padding: 10px !important;
    width: 40%;
    margin: auto !important;
    margin-top: 20px !important;
    margin-bottom: 20px !important;
    min-height: 600px;
    max-height: 600px;
    color: white;
    overflow: auto;
  }

  #return {
    position: absolute;
    background: black;
    border-radius: 5px;
    opacity: 85%;
    width: 40%;
    right: 50px;
    bottom: 122px;
    color: white;
    font-size: 20px;
  }
}