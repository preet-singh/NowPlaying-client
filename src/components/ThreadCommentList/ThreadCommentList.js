import React from 'react';
import './ThreadCommentList.css';
import ThreadCommentItem from '../ThreadCommentItem/ThreadCommentItem';

function ThreadCommentList(props) {
  return (
    <div className='thread-comment-list'>
      <ThreadCommentItem />
      <ThreadCommentItem />
      <ThreadCommentItem />
    </div>
  )
}

export default ThreadCommentList;