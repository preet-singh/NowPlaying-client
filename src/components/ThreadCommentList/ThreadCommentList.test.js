import React from 'react';
import ThreadCommentList from './ThreadCommentList.js';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe(`ThreadCommentList component`, () => {

  it('renders the ThreadCommentList', () => {
    const wrapper = shallow(<ThreadCommentList />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

})