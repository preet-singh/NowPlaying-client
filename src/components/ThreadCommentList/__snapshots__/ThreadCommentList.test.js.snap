// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThreadCommentList component renders the ThreadCommentList 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
